partner_contact_personal_information_page/__manifest__.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
```

Comment:
Do away with coding lines. And while you are on headers, change the licence lines to https: